docs/guide/logger.rst

```diff
@@ -0,0 +1,64 @@
+.. _logger:
+
+**************
+Logging system
+**************
+
+Overview
+========
+
+The Sunpy logging system makes use of the :ref:`Astropy loggin system <astropy:logging>`.
```

Comment:
There is a typo:
```suggestion
The Sunpy logging system makes use of the :ref:`Astropy logging system <astropy:logging>`.
```

Also this link dosen't work and causes the doc build to fail.

Reply:
fixed.